src/core/constants.js

```diff
@@ -190,17 +190,34 @@ export const UNINSTALL_COMPLETE = 'UNINSTALL_COMPLETE';
 export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking install actions.
+export const ENABLE_ACTION = 'enable';
 export const INSTALL_ACTION = 'install';
+export const INSTALL_CANCELLED_ACTION = 'install:cancelled';
+export const INSTALL_ERRORED_ACTION = 'install:errored';
+export const INSTALL_FAILED_ACTION = 'install:failed';
 export const INSTALL_STARTED_ACTION = 'install:started';
 export const UNINSTALL_ACTION = 'uninstall';
 
 // Tracking Event Categories.
 // WARNING: Do not change these without notifying data + metrics teams.
 // Changing these strings will break existing statistics without
 // updating the category matching at the same time.
+export const ENABLE_THEME_CATEGORY = 'AMO Theme Activations';
+export const ENABLE_EXTENSION_CATEGORY = 'AMO Addon Activations';
+
 export const INSTALL_THEME_CATEGORY = 'AMO Theme Installs';
 export const INSTALL_EXTENSION_CATEGORY = 'AMO Addon Installs';
 
+export const INSTALL_THEME_CANCELLED_CATEGORY = 'AMO Theme Installs Cancelled';
+export const INSTALL_EXTENSION_CANCELLED_CATEGORY =
```

Comment:
I am wondering it's better to have the more specific description first followed by the type (theme vs extension):
INSTALL_CANCELLED_THEME_CATEGORY
INSTALL_CANCELLED_EXTENSION_CATEGORY

INSTALL_ERRORED_THEME_CATEGORY
INSTALL_ERRORED_EXTENSION_CATEGORY (etc)

Reply:
also - not sure since how it is important to notify anyone about these updates since we are not technically changing the existing ones, we are just adding new ones..

Reply:
You mean re-sorting the constants?

Reply:
I just meant renaming them a bit. I might be overthinking this : ) but just thought it might be clearer when you first look at the code to see it the way I have above. 

Immediately you can see INSTALL_CANCELLED_[TYPE] or INSTALL_ERROR_[TYPE] vs the theme or extension breaking up the description (IMO). If that doesn't makes any sense or you disagree that it's more obvious this way, then just disregard. It was just a "thinking out loud" suggestion :)

Reply:
ah right, makes sense